backend/tests/login.test.js

```diff
@@ -43,13 +47,15 @@ describe("POST /login", () => {
 
     it("should return 200 for successful authentication", async () => {
         userEmailExists.mockResolvedValue(true);
-        User.findOne.mockResolvedValue({ email: "test@test.com", password: "hashedPassword123" });
+        User.findOne.mockResolvedValue({_id: "userid", email: "test@test.com", password: "hashedPassword123" });
```

Comment:
not sure if this change was intentional? cuz on login.js it seems like the logic didnt' change